<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadDataset</class>
 <widget class="QDialog" name="LoadDataset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Bids Dataset</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Path">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>22</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dataset path:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPath">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxGI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>481</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Dataset General information</string>
   </property>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DatasetType: </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDOI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DatasetDOI:</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollAreaAuth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>459</width>
       <height>219</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoadDataset</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoadDataset</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
